# AudioCustomMsg.msg
int16[] rearLeft
int16[] rearRight
int16[] frontLeft
int16[] frontRight